Data/ODBC/testsuite/src/ODBCSQLServerTest.cpp

```diff
@@ -417,6 +417,30 @@ catch(Poco::Exception& ex)
 }
 }
 
+void ODBCSQLServerTest::testUUIDsBulk()
+{
+	try
+	{
+		if (!_pSession) fail("Test not available.");
+
+		_pSession->setFeature("autoBind", true);
+		_pSession->setFeature("autoExtract", true);
+
+		recreateUUIDsTable();
+		int rows = 1000;
+		std::vector<Poco::UUID> uuids(rows);
+		for (int i = 0; i < rows; ++i) {
+			uuids[i]= Poco::UUIDGenerator::defaultGenerator().createRandom();
+		}
+
+		*_pSession << "INSERT INTO Strings VALUES (?)"s, use(uuids, bulk), Poco::Data::Keywords::now;
```

Comment:
what does this exactly do? Please make sure your PR compiles and executes the test (which should actually test the mew functionality).